src/main/java/games/strategy/triplea/ai/proAI/ProTechAI.java

```diff
@@ -737,15 +737,6 @@ private static Route getMaxSeaRoute(final GameData data, final Territory start,
    * Assumes that water is passable to air units always.
    */
   private static Match<Territory> TerritoryIsImpassableToAirUnits() {
-    return new Match<Territory>() {
-      @Override
-      public boolean match(final Territory t) {
-        if (Matches.TerritoryIsLand.match(t) && Matches.TerritoryIsImpassable.match(t)) {
-          return true;
-        }
-        return false;
-      }
-    };
+    return Match.all(Matches.TerritoryIsLand, Matches.TerritoryIsImpassable);
```

Comment:
This is not just an extraction.  Please scrutinize to ensure the logic is identical.